addons/web/static/src/scss/dropdown.scss

```diff
@@ -9,6 +9,7 @@
     font-size: $font-size-base;
     padding: 5px 0px;
     box-shadow: 0px 6px 12px rgba(0, 0, 0, 0.176);
+    word-wrap: break-word;
```

Comment:
`ui-autocomplete` is used by other modules (e.g. website), and this likely might break current styling.
I prefer to play safe and apply this style in the custom class name that is used specifically in discuss code.